sshh/main.py

```diff
@@ -110,7 +110,8 @@ def cmd_agent(request):
         sshenv['PS1'] = f"[{request.group}]{sshenv['PS1']}"
         logger.info('ssh-agent PID=%s session "%s" has been started. To close this session, exit shell.',
                     sshenv['SSH_AGENT_PID'], request.group)
-        subprocess.run (sshenv['SHELL'], env=sshenv)
+        bash_command = f'{sshenv["SHELL"]} --init-file <(echo "export PS1={sshenv["PS1"]}"'
```

Comment:
COULD:

Only execute with `-init-file` if PS1 setting from `sshenv` doesn't take

I'm not sure how to do that without first executing subprocess though, and I think this solution should work on any unix system.

Perhaps set it only if `sshenv["SHELL"]` is actually executing `bash` and not something else